lib/order_query/point.rb

```diff
@@ -32,19 +32,19 @@ def position
     end
 
     # @return [ActiveRecord::Relation]
-    def after
-      side :after
+    def after(strict = true)
+      side :after, strict
     end
 
     # @return [ActiveRecord::Relation]
-    def before
-      side :before
+    def before(strict = true)
```

Comment:
How about making `strict` a keyword argument and bumping the `required_ruby_version` in gemspec to `~> 2.1`?